modules/updateLists.js

```diff
@@ -0,0 +1,19 @@
+import removeListitem from './removeListitem.js';
+import displayLists from './displayLists.js';
+
+export default function updateLists(toDoList) {
+  displayLists(toDoList);
+
+  const removeTaskIcons = document.querySelectorAll('.list-icon');
+  function handleRemoveTaskClick(event) {
+    const listItem = event.target.closest('li');
+    const index = listItem.getAttribute('data-index');
+    removeListitem(index, toDoList);
+    // updateLists(toDoList);
```

Comment:
- [OPTIONAL] Kindly remove this commented line of code so as to make your code tidy.